fdb-relational-jdbc/src/main/java/com/apple/foundationdb/relational/jdbc/ParameterHelper.java

```diff
@@ -40,55 +39,48 @@ public class ParameterHelper {
     public static Parameter ofBoolean(boolean b) {
         return Parameter.newBuilder()
                 .setType(Type.BOOLEAN)
-                .setJavaSqlTypesCode(Types.BOOLEAN)
```

Comment:
It would seem to me that for cross-version compatibility reasons, we'd need to keep setting this field at least for one minor version. Otherwise, I'm not sure how old code would make sense of the `Parameter`'s type. Is my understanding wrong, or are we not covering this in mixed mode tests? Or something else?